MSAL-Legrand/0.4.2/MSAL-Legrand.podspec

```diff
@@ -0,0 +1,51 @@
+Pod::Spec.new do |s|
+  s.name         = "MSAL-Legrand"
+  s.version      = "0.4.2"
```

Comment:
You will want a different version than the current pod.

Reply:
Changed to 0.4.2.1